src/lazyload-image.ts

```diff
@@ -110,16 +111,16 @@ function setLoadedStyle(element: HTMLImageElement | HTMLDivElement) {
     const styles = element.className
         .split(' ')
         .filter(s => !!s)
-        .filter(s => s !== 'ng-lazyloading');
-    styles.push('ng-lazyloaded');
+        .filter(s => s !== cssClassNames.loading);
+    styles.push(cssClassNames.loaded);
     element.className = styles.join(' ');
     return element;
 }
 
 export function lazyLoadImage(element: HTMLImageElement | HTMLDivElement, imagePath: string, defaultImagePath: string, errorImgPath: string, offset: number, useSrcset: boolean = false, scrollContainer?: HTMLElement) {
     setImageAndSourcesToDefault(element, defaultImagePath, useSrcset);
-    if (element.className && element.className.includes('ng-lazyloaded')) {
-        element.className = element.className.replace('ng-lazyloaded', '');
+    if (element.className && element.className.includes(cssClassNames.loaded)) {
+        element.className = element.className.replace(cssClassNames.loaded, '');
```

Comment:
This is much better but I still think that we should create a function that is only responsible for setting and removing css class names. Something like:
```ts
function removeCssClassName(element: HTMLImageElement | HTMLDivElement, cssClassName: string) {
  element.className = element.className.replace(cssClassName, '');
}

function addCssClassName(element: HTMLImageElement | HTMLDivElement, cssClassName: string) {
  if (!element.className.includes(cssClassName)) {
    element.className += `  ${cssClassName}`
  }
}
```

And than use these two function instead of:
```ts
if (element.className && element.className.includes(cssClassNames.loaded)) {
  element.className = element.className.replace(cssClassNames.loaded, '');
}
```  
```ts
element.className += ' ' + cssClassNames.failed;
```
```ts
setLoadedStyle
```

Reply:
@rimlin, I saw your comment now, regarding that you addressed 2, 4, 5 and not 3 which this comment is about. I will therefore accept this PR without this change.